tests/google_sheets_2/test_google_sheets_2.py

```diff
@@ -186,7 +187,7 @@ def mock_api_responses(uri: str):
     )
     assert (
         fetch_mock.call_args_list[1][0][0]
-        == 'https://sheets.googleapis.com/v4/spreadsheets/1SMnhnmBm-Tup3SfhS03McCf6S4pS2xqjI6CAXSSBpHU/values/Foo?valueRenderOption=UNFORMATTED_VALUE'
+        == 'https://sheets.googleapis.com/v4/spreadsheets/1SMnhnmBm-Tup3SfhS03McCf6S4pS2xqjI6CAXSSBpHU/values/Foo?valueRenderOption=UNFORMATTED_VALUE&DateTimeRenderOption=FORMATTED_STRING'
```

Comment:
As per doc, this can depend on the spreadhseet locale (https://developers.google.com/sheets/api/reference/rest/v4/DateTimeRenderOption)
We must be aware this will probably break if the locale is not `en`. This is OK but we should plan on adding it to the docs. :)